server/routes/offlinePayment.routes.js

```diff
@@ -93,4 +102,97 @@ router.put(
  */
 router.get('/all', authenticate, isUserOrAdmin, OfflinePaymentController.getAll);
 
+/**
+ * @swagger
+ * /offline-payment/resolve/:id:
+ *   put:
+ *     tags:
+ *       - OfflinePayment
+ *     description: Resolves an offline payment
+ *     produces:
+ *       - application/json
+ *     requestBody:
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/schemas/OfflinePayment'
+ *      description: Resolves an offline payment
+ *     responses:
+ *      '200':
+ *        description: Offline payment resolved
+ *      '400':
+ *        description: Error resolving offline payment
+ *      '500':
+ *       description: Internal server error
+ */
+router.put(
+  '/resolve/:id',
+  authenticate,
+  hasValidObjectId,
+  isAdmin,
+  OfflinePaymentController.resolveOfflinePayment,
+);
+
+/**
+ * @swagger
+ * /offline-payment/comment:
+ *   put:
+ *     tags:
+ *       - OfflinePayment
+ *     description: Raise a comment for a particular offline payment
+ *     produces:
+ *       - application/json
+ *     requestBody:
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/schemas/OfflinePayment'
+ *      description: Raise a comment for a particular offline payment
+ *     responses:
+ *      '200':
+ *        description: Comment raised
+ *      '400':
+ *        description: Error raising comment
+ *      '500':
+ *       description: Internal server error
+ */
+router.put(
+  '/comment',
```

Comment:
I think we should be explicit here so that it is easier to differentiate when raising or resolving a comment.
```suggestion
  '/raise-comment',
```